### PR TITLE
Bugfix: include <> -> include ""

### PR DESCRIPTION
**Pull request by kornelu123**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
+
+add_executable(${PROJECT_NAME}
+  main.cpp
+  mt_rng.cpp
+)
+
+target_include_directories(ble_sim PRIVATE
```
Comment (husxk) [inline CMakeLists.txt]: ble_sim -> ${PROJECT_NAME}

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(ble_sim C CXX ASM)
+
+set(CMAKE_C_STANDARD 11)
```
Comment (husxk) [inline CMakeLists.txt]: Do not need to set C ASM.
We can add it later, if we need to do so.

File: Makefile
Comment (husxk) [inline Makefile:1]: Please add `.PHONY` for `all ninja clean`

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+all: ninja build
+
+build:
```
Comment (husxk) [inline Makefile]: Please change `build` to `build/` to highlight that it is a directory

**Review by husxk — CHANGES_REQUESTED** [review]
Good changes, although few comments

File: CMakeLists.txt (hunk 2 above)
Comment (kornelu123) [reply CMakeLists.txt]: I also removed C standard from CMakeLists.txt.

**Review by husxk — APPROVED** [review]
Thanks!

**Pull request merged**